03-getting-started.Rmd

```diff
@@ -3,11 +3,56 @@
+
+Expression Data (from RNA-Seq):
+Quantitative data on gene expression levels, showing how much of each gene is transcribed into RNA.
+
+
+To sum up the applications of sequencing data here are the different applications 
+Genomics: Studying entire genomes to understand genetic variation, evolutionary relationships, and disease mechanisms.
+
+Transcriptomics: Analyzing RNA sequences to study gene expression patterns, identify novel transcripts, and understand regulatory mechanisms.
+
+Metagenomics: Investigating genetic material from environmental samples to study microbial communities and their functions.
+
+Personalized Medicine: Using genetic information to tailor medical treatments to individual patients based on their unique genetic makeup.
+
 ## What is PII and PHI and why is it important? Why is genomic data often protected?
+PII (Personally Identifiable Information): This refers to any information that can be used to identify an individual. Examples include names, social security numbers, addresses, phone numbers, email addresses, and other personal details. PII is important because unauthorized access or disclosure of this information can lead to identity theft, fraud, and other forms of exploitation.
+
+
+PHI (Protected Health Information): This is a subset of PII specific to the healthcare industry. It includes any health information that can be linked to a specific individual, such as medical records, health history, treatment plans, and other health-related data. PHI is crucial because improper handling of this information can result in breaches of patient confidentiality, misuse of sensitive health data, and violations of privacy rights.
+
+Protecting PII (Personally Identifiable Information) and PHI (Protected Health Information) is crucial for several reasons. Both types of information contain sensitive details about individuals, and safeguarding them helps maintain privacy while preventing unauthorized access or misuse. For instance, PII, such as social security numbers or financial data, can be exploited for identity theft or fraud, making its protection essential in reducing these risks. PHI includes sensitive health data that, if disclosed improperly, could result in stigma, discrimination, or personal distress, thus protecting PHI is key to preserving individuals' dignity and autonomy. The compliance with laws and regulations like GDPR in Europe and HIPAA in the U.S. requires robust protection of PII and PHI, with non-compliance potentially leading to significant legal and financial repercussions. Handling this information responsibly also fosters trust with customers, clients, and patients, which is vital for maintaining strong relationships and a positive reputation. Additionally, safeguarding PII and PHI helps defend against unauthorized access and cyber threats, ensuring that sensitive data remains secure from malicious actors.
```

Comment:
Have you been using AI to write any of this? If so, that's totally fine, but we do need to cite what AI tool was used. Let me know.